meshes/terrain-package.js

```diff
@@ -0,0 +1,60 @@
+import * as THREE from 'three';
+import { EXRLoader } from 'three/examples/jsm/loaders/EXRLoader.js';
+
+const exrLoader = new EXRLoader();
+const textureLoader = new THREE.TextureLoader();
+
+export const DIFFUSE_MAP = 'diffuse';
+export const NORMAL_MAP = 'normal';
+export const NOISE_MAP = 'noise';
+export const ENV_MAP = 'env';
+
+class TerrainPackage {
+  constructor(textures) {
+    this.textures = textures;
+  }
+
+  get data() {
+    return this.textures;
+  }
+
+  static async loadUrls(diffNames, normalNames, envName, noiseName) {
```

Comment:
Might be better to do these by options argument rather than positional arguments.

Reply:
Yeah, I agree